testing/steps/neuraxle_test_case.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
```

Comment:
don't forget apache 2 headers in all unit tests files of the project. I'm not sure if tests needs a name with a `======` title.